lib/states/ks/wichita.R

```diff
@@ -58,14 +58,17 @@ clean <- function(d, helpers) {
         "pedestrian",
         "vehicular"
       ),
+      citation_date_time = sub("\\+.*", "", citation_date_time), 
```

Comment:
Can you add a comment explaining what's going on here?

Reply:
see comment added